WalletWasabi/WabiSabi/Client/CoinJoinTrackerFactory.cs

```diff
@@ -29,7 +29,8 @@ public CoinJoinTracker CreateAndStart(Wallet wallet, IEnumerable<SmartCoin> coin
 			new KeyChain(wallet.KeyManager),
 			new InternalDestinationProvider(wallet.KeyManager),
 			RoundStatusUpdater,
-			wallet.ServiceConfiguration.MinAnonScoreTarget);
+			wallet.ServiceConfiguration.MinAnonScoreTarget,
+			doNotRegisterInLastMinuteTimeLimit: TimeSpan.FromMinutes(1));
```

Comment:
It's hard to believe but I cannot find a good name for this parameter, bet i am sure there has to be one.

Reply:
`prohibitedRegistrationTimeLimit`?

Reply:
Let poets find a better word, we only need a functional name :)